pkg/apis/kops/v1alpha1/zz_generated.conversion.go

```diff
@@ -1,7 +1,7 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright 2017 The Kubernetes Authors.
+Copyright 2018 The Kubernetes Authors.
```

Comment:
Can you rebase, and then probably want to re-run make apimachinery.  I think these changes will be removed.

Reply:
Done